packages/core/src/services/zoom.ts

```diff
@@ -80,4 +115,37 @@ export class Zoom extends Service {
 			"zoomRatio"
 		);
 	}
+
+	// filter out data not inside zoom domain
+	// to get better range value for axis label
+	filterDataForRangeAxis(displayData: object[], configs?: any) {
+		const zoomDomain = this.model.get("zoomDomain");
+		const mergedConfigs = Object.assign(
+			{ stacked: false }, // default configs
+			configs
+		);
+		const shouldUpdateRangeAxis = Tools.getProperty(
+			this.model.getOptions(),
+			"axes",
+			this.services.cartesianScales.getMainYAxisPosition(),
+			"updateWhenZooming"
```

Comment:
I was just testing this and realized that `updateWhenZooming` is available for all axes, however it only takes effect for the main Y-axis. this means that I can set it to false for the main X-axis and it will do nothing

Reply:
we should either just make it available for the range axis like before, or if we are allowing it to be used for all axis then it should achieve something when set to false

Reply:
@theiliad The domain axis is always changing while zoom domain is changing, so no option is required.
This new option should only works for the range axis (main Y axis).
So I change it back to  `updateRangeAxis` under `zoomBar`.

Updated in https://github.com/carbon-design-system/carbon-charts/pull/766/commits/2cd37ad112dcf86e38faebf6534fb397ed5225f4.

Please let me know if this is not what you expected.